zstacklib/zstacklib/utils/sanlock.py

```
@@ -447,47 +554,91 @@ def get_lockspace(vg_uuid):
+        os.mkdir(SANLOCK_BACKUP_PATH)
+
+    bk_file = os.path.join(SANLOCK_BACKUP_PATH, "%s.lockspace" % vg_uuid)
+    r, o, e = read_delta_lease(vg_uuid, host_id)
+    if r != 0:
+        return
+    context = {}
+    meta_backup = linux.read_file(bk_file)
+    if meta_backup:
+        try:
+            context = jsonobject.loads(meta_backup).__dict__
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)